polylogue/services/conversation_registrar.py

```
@@ -63,6 +63,18 @@ def record_document(
             return
 
+        existing_branch: Optional[str] = None
+        try:
+            with open_connection(self.database.resolve_path()) as conn:
+                row = conn.execute(
+                    "SELECT current_branch FROM conversations WHERE provider = ? AND conversation_id = ?",
+                    (provider, conversation_id),
+                ).fetchone()
+                if row is not None:
+                    existing_branch = row["current_branch"]
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)